integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/convert_currency/model/BaseIdForFaitDataUIState.kt

```diff
@@ -0,0 +1,10 @@
+package org.dash.wallet.integration.coinbase_integration.ui.convert_currency.model
+
+import org.dash.wallet.common.ui.payment_method_picker.PaymentMethod
+import org.dash.wallet.integration.coinbase_integration.model.BaseIdForUSDData
+
+sealed class BaseIdForFaitDataUIState {
```

Comment:
perhaps, `Fait` should be `Fiat` -- we can fix this in a different PR if that is the case.